code/modules/mapfluff/ruins/spaceruin_code/TheDerelict.dm

```diff
@@ -191,7 +191,7 @@
 		"\[00:01\] Every single day I spend on this station",
 		"\[00:07\] I become more and more certain that this was some kind of macabre punishment assignment",
 		"\[00:17\] I don't know what I did to deserve being consigned to this glued together ball of orbital wreckage",
-		"\[00:28\] But someones got it out for me",
+		"\[00:28\] But someone has it out for me",
```

Comment:
This one isn't an error, just a variant. https://idioms.thefreedictionary.com/have+got+it+in+for+(one)
Should be `someone's got it out for me`